examples/linear_static.jl

```diff
@@ -0,0 +1,104 @@
+# This file is a part of JuliaFEM.
+# License is MIT: see https://github.com/JuliaFEM/JuliaFEM.jl/blob/master/LICENSE.md
+
+# # Linear Static Example
+
+# ![](linear_static/freecad.png)
+
+# ## Preprocessing
+using JuliaFEM
+using JuliaFEM.Preprocess
+using JuliaFEM.Postprocess
+add_elements! = JuliaFEM.add_elements!
+
+# First we will read in the mesh. Geometry and mesh are greated with FreeCAD,
+# where med format is selected for exporting. Mesh file consist also edge and
+# surface mesh, which we will need to neglect later.
+
+datadir = Pkg.dir("JuliaFEM", "examples", "linear_static")
+meshfile = joinpath(datadir, "JuliaFEMSMP18.med")
+mesh = aster_read_mesh(meshfile)
+
+# Next we will create the model and define Elasticity. Also elements are added
+# to the model.
+
+model = Problem(Elasticity, "OTHER", 3)
+model_elements = create_elements(mesh, "OTHER")
+
+# Elements need material properties and they are defined next
+
+update!(model_elements, "youngs modulus", 208.0E3)
+update!(model_elements, "poissons ratio", 0.30)
+update!(model_elements, "density", 7.80E-9)
+JuliaFEM.add_elements!(model, model_elements)
+
+# Let's ingore Seg3 and Tri6 elements using `filter`
+model.elements = collect(filter(x -> !isa(x, Union{Element{Seg3},
+                                            Element{Tri6}}), model.elements));

```

Comment:
If you want to talk about multiple dispatch, here's your line. Another way to do this, maybe "more Julian" way:

```julia
is_not_Seg3_or_Tri6(::Element{T}) where {T<:Union{Seg3, Tri6}} = false
is_not_Seg3_or_Tri6(::Element) = true
elements = [Element(Seg2, [1, 2]), Element(Seg3, [1, 2, 3]), Element(Quad4, [1, 2, 3, 4])]
collect(filter(is_not_Seg3_or_Tri6, elements))
```

Reply:
I'm not against making it to function, but this doesn't make this any clearer. I don't understand what this does: 
```julia
elements = [Element(Seg2, [1, 2]), Element(Seg3, [1, 2, 3]), Element(Quad4, [1, 2, 3, 4])] collect(filter(is_not_Seg3_or_Tri6, elements))
````